### ISSUE TITLE
Concatenation | Keras | Tensorflow backend

### ISSUE THREAD

**Issue by SwagatDas** [keras-team/keras#12975]
What should I do, for the following Value Error:

"A `Concatenate` layer requires inputs with matching shapes except for the concat axis. Got inputs shapes: [(1024,), (1024, 2)]"

I want the final output tensor to be of shape: [1024, 3]

**Comment by SwagatDas** [comment]
Got it. I was supposed to use:
tensorflow.reshape(tensor, shape=[-1, 1])
and then the concatenation axis as 1.
